b2b_ecommerce/api.py

```diff
@@ -26,10 +26,20 @@ def complete_b2b_order(order):
     Args:
         order (B2BOrder): A fulfilled order for enrollment codes
     """
+
+    if order.coupon and order.contract_number:
+        name = f"{order.contract_number} {order.coupon.coupon_code}"
+    elif order.contract_number:
+        name = order.contract_number
+    elif order.coupon:
+        name = order.coupon.coupon_code
+    else:
+        name = f"CouponPayment for order #{order.id}"
```

Comment:
Maybe simplify to `#{order.id}`?

Reply:
I think we will basically never not have either a b2b coupon or contract number